pkg/cmd/chat/utils/fileupload.go

```diff
@@ -8,22 +8,56 @@ import (
 	"github.com/spf13/cobra"
 )
 
-func UploadFile(c *stream.Client, cmd *cobra.Command, chType, chId, userID, filePath string) (string, error) {
+type uploadType string
+
+const (
+	uploadTypeFile  uploadType = "file"
+	uploadTypeImage uploadType = "image"
+)
+
+func UploadFile(c *stream.Client, cmd *cobra.Command, chType, chId, userID, filePath, contentType string) (string, error) {
+	return uploadFileInternal(c, cmd, uploadTypeFile, chType, chId, userID, filePath, contentType)
```

Comment:
I don't think we need `Internal` suffix

Reply:
oh yeah, golang won't complain about it. fixed. ✅